components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -1160,6 +1183,34 @@ data class SearchResult(
     }
 }
 
+/**
+ * Information about a top frecent site. Returned by `PlacesAPI.getTopFrecentSiteInfos`.
+ */
+data class TopFrecentSiteInfo(
+    /**
+     * The frecency.
+     */
+    val frecency: Long,
```

Comment:
I think this is OK to expose to a-c, though. The cat's out of the bag because we _do_ expose frecency in autocomplete results, which a-c [renames](https://github.com/mozilla-mobile/android-components/blob/cb312ff60cc8df65b22edbf9f56e099ccd4e89ba/components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesHistoryStorage.kt#L103) to "score" and doesn't make any guarantees beyond ["higher = more relevant"](https://github.com/mozilla-mobile/android-components/blob/cb312ff60cc8df65b22edbf9f56e099ccd4e89ba/components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt#L188-L189).

Reply:
I am gonna remove this for the time being since I don't feel I will need this in the short term.